java-frontend/src/test/java/org/sonar/java/model/declaration/ClassTreeImplTest.java

```diff
@@ -100,12 +101,16 @@ void records_members_order() {
     assertThat(classTree).is(Tree.Kind.RECORD);
     List<String> membersKinds = classTree.members().stream().map(Tree::kind).map(Tree.Kind::name).collect(Collectors.toList());
     assertThat(membersKinds).containsExactly(
+      "CONSTRUCTOR",
       "VARIABLE",
       "METHOD",
       "VARIABLE",
       "CONSTRUCTOR",
       "VARIABLE",
       "CLASS",
       "VARIABLE");
+
+    MethodTreeImpl constructor = (MethodTreeImpl) classTree.members().get(0);
+    assertThat(constructor.getLine()).isEqualTo(2);
```

Comment:
I think this assertion should be tested in a dedicated test

Reply:
We could probably move this to `MethodTreeImplTest.java`

Reply:
Seemed also weird to me, but did not know where to put it.

Reply:
:+1: